src/TTrigger.cpp

```diff
@@ -240,18 +236,10 @@ bool TTrigger::setRegexCodeList( QStringList regexList, QList<int> propertyList
              }
              else
              {
-                 if( mudlet::debugMode )
```

Comment:
Have taken this message back out to keep the behaviour like it was before (I think).